db/seeders/users_seeder.rb

```diff
@@ -0,0 +1,29 @@
+class UserSeeder
```

Comment:
This class is named in the singular, `UserSeeder`, and that makes sense because it is creating only 1 user. (Why is that, by the way?) But the filename is `users_seeder.rb`. Should that be changed?

Reply:
I think keep all in plural will be better.

Reply:
Since you are agreeing to move the user creations into this file, plural will be appropriate. I suggest changing the class name to reflect that as well, to be consistent with the others.